<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui1</class>
 <widget class="QMainWindow" name="gui1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Acerte se puderes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_comecar">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>400</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #075ede;
	color: #f5f5f5;
	border: 1px solid transparent;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: #094aaa;
}</string>
    </property>
    <property name="text">
     <string>Iniciar o jogo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string>Acerta Se Puderes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_port">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid #666;
	border-radius: 6px;
	color: rgb(245, 245, 245);
	padding-left: 10px;
	outline-color: #fff;
}
QLineEdit:focus {
	border: 1px solid #939393;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="in_nome">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>260</y>
      <width>400</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid #666;
	border-radius: 6px;
	color: rgb(245, 245, 245);
	padding-left: 10px;
}
QLineEdit:focus {
	border: 1px solid #939393;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>Nome do jogador</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_port_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>Porta Serial</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_img">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>291</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/arduino.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
